mod_auth_openidc_alpine (#1)
r2d2bzh/docker-oidc-rp-apache: README.md

[![Code Quality: Cpp](https://img.shields.io/lgtm/grade/cpp/g/zmartzone/mod_auth_openidc.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/zmartzone/mod_auth_openidc/context:cpp)

> [![Total Alerts](https://img.shields.io/lgtm/alerts/g/zmartzone/mod_auth_openidc.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/zmartzone/mod_auth_openidc/alerts)

Reviewer: If we want to keep these, they need to be updated to point to our repository.

Reviewer: done

mod_auth_openidc - alpine
=========================

> This project is a fork of the mod_auth_openidc project. 

Reviewer: ```suggestion
This project is a fork of [zmartzone/mod_auth_openidc](https://github.com/zmartzone/mod_auth_openidc). 
```

The key differences:

> - The dockerfile builds from an alpine image by default instead of debian.

Reviewer: ```suggestion
- The Alpine image is used by default instead of Debian.
```

> - The debian option has been removed.

Reviewer: ```suggestion
- The Debian option has been removed.
```
